#request
float64 x
float64 y
float64 z
float64 oW
float64 oX
float64 oY
float64 oZ
---
#response
bool status